src/mmw/apps/modeling/views.py

```diff
@@ -229,6 +242,9 @@ def _initiate_tr55_job_chain(model_input, job_id):
 
 
 def _construct_tr55_job_chain(model_input, job_id):
+    exchange = 'C.dq'
```

Comment:
A comment to that effect would be good. I also think if it is something non-arbitrary and somewhat magic, it probably should be a constant.